src/models/SearchResultsBib.ts

```diff
@@ -0,0 +1,81 @@
+import type { SearchResult } from "../types/searchTypes"
+import type { ElectronicResource } from "../types/bibTypes"
+
+export default class SearchResultsBib {
```

Comment:
The class name is pretty descriptive already but can you add a comment overview about the changes. It's a bit hard because some reasons for the business logic, such as taking the first item in the `publicationStatement` array, isn't documented anywhere (as far as I know). That may be a good question for Brooke or Sean?

Reply:
the majority if not all properties that only refer to the first element follow this pattern:
```
 "dateEndString": [
    "1990"
  ],
```
where there is a single-element array (due to JSON LD and or Elastic Search requirements). If there is a place where the expected interface of the api response is defined, it may be worth creating a type that denotes a single element array.

Reply:
I don't think this is possible in typescript or may involve some intricate type definitions. Worth looking into but we could define the pattern in a general "how the API properties work"?

Reply:
I added a comment above SearchResultsBib with some context as well as a note about extracting single-item arrays in the constructor. I think it makes to have some data formatting in the class constructor, so not sure it's necessary to add a section to the main docs about this, but happy to add one if you think it would be helpful to have a high-level overview of this pattern.